cypress/integration/installation.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { Epinio } from '~/cypress/support/epinio';
+import { TopLevelMenu } from '~/cypress/support/toplevelmenu';
+
+Cypress.config();
+describe('Epinio installation testing', () => {
+  const topLevelMenu = new TopLevelMenu();
+  const epinio = new Epinio();
+
+  beforeEach(() => {
+    cy.login();
+    cy.visit('/home');
+    topLevelMenu.openIfClosed();
+    cy.get('.clusters').contains('local').click()
+  });
+
+  it('Add the Epinio helm repo', () => {
+    cy.addHelmRepo({repoName: 'epinio-repo', repoUrl: 'https://epinio.github.io/helm-charts'});
```

Comment:
Maybe better to have a variable here? Like for the domain or the Rancher URL.

Reply:
Yes I agree, I will add the function with the uninstallation test.

Reply:
The var you mean I think :)